src/Discord.Net.WebSocket/Entities/Channels/SocketVoiceChannel.cs

```diff
@@ -11,10 +11,13 @@
 namespace Discord.WebSocket
 {
     [DebuggerDisplay(@"{DebuggerDisplay,nq}")]
-    public class SocketVoiceChannel : SocketGuildChannel, IVoiceChannel, ISocketAudioChannel
+    public class SocketVoiceChannel : SocketGuildChannel, IVoiceChannel, ISocketAudioChannel, INestedChannel
```

Comment:
Redundant inheritance since `IVoiceChannel` should has it.
(Forgot to make `IVoiceChannel` inherit `INestedChannel`)